tests/network/sriov.go

```diff
@@ -86,26 +86,6 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 
 	sriovResourceName := readSRIOVResourceName()
 
-	createSriovNetworkAttachmentDefinition := func(
-		networkName, namespace string, vlanID int, enableLinkState bool,
-	) error {
-		const pluginType = "sriov"
-		pluginConf := map[string]interface{}{"vlan": vlanID}
-		if enableLinkState {
-			pluginConf[linkStateKey] = linkStateEnableVal
```

Comment:
in commit: `tests, net: Add a SR-IOV network-attachment-definition helper` please also remove the linkStateEnabled, linkStateKey and linkStateEnableVal consts as they are no longer used.